src/input/input_python.rs

```diff
@@ -116,6 +116,10 @@ impl<'a> Input<'a> for PyAny {
         }
     }
 
+    fn input_is_python(&self) -> Option<&PyAny> {
```

Comment:
Seems like this should be `input_to_python` or something like that since it doesn't returns a boolean.